src/main/java/com/sarality/db/TableDefinition.java

```diff
@@ -15,6 +20,8 @@
   private final String tableName;
   // The version of the database defined by the given set of columns
   private final int tableVersion;
+  // A list of updates for each version
+  private Map<Integer, Map<UpdateOperationType, List<UpdateOperation>>> versionUpdates = new HashMap<>();
```

Comment:
Do we really need to do this processing in the TableDefinition. All this should have in a List of SchemaUpdate

The Map may need to be in a utility class like a SchemaUpdater, which in turn will reply on different SchemaUpdateSqlGenerators